### ISSUE TITLE
Rebalance Martial Arts and Brawl

### ISSUE THREAD

**Issue by noeinan** [noeinan/DoL-World-Expansion-Mod#85]
Many players have complained these are too powerful and need to be readjusted. 

I want max brawl and max martial arts together to basically be a oneshot kill. 

However, brawl is meant to be weaker than martial arts. And people can choose not to train MA but can't choose not to train brawl if they are defiant. I should weaken brawl at the very least.

**Comment by Andrest07** [comment]
If I may suggest, martial arts in the real world USUALLY teaches you to only fight against one person and to run away if you're outnumbered but obviously, in this game you can't do that. Obviously master martial artists would be able to fight against maybe two or three but never like in the movies. What I suggest is that for every additional NPC involved in a fight, martial arts' effectiveness would be cut by 20%, so from 100% to 80%, to 64%, to 51.2% and so on. Yes this keep 1 on 1 fights really easy but in group fights? Well, you're in trouble.

Brawling however is an unorthodox fighting style that is learned purely through experience and most times, brawls include multiple people so brawling isn't affected by the debuff that martial arts would get from multiple opponents. On the other hand however, although it has this advantage, when compared against martial arts in a 1 on 1 fight, it will always lose so as you mentioned, it's baseline effect would be weaker compared to martial arts despite having the same effectiveness against multiple people.

TLDR;
Make it so martial arts get weaker the more people there are while brawling is always weaker than martial arts but stays effective even against multiple enemies.

**Comment by Andrest07** [comment]
Also in my opinion, brawling as it is isn't OP? I'd have to test it on a separate character but I play exclusively consensual and if it's not, always defiant. With beauty at max and allure close to max at day and max at night, this means a lot of fights for me but despite that I still only have D+brawling. Stats-wise I have hit 499 people, so yeah.

**Comment by noeinan** [comment]
I really like the idea of Martial Arts having lowered effectiveness against more enemies. Maybe we can eventually add a new skill "gain opportunity" or something, which if you pass a successful martial arts check then you can escape the encounter?

Agreed to Brawling being more unorthodox. I had originally split in two bc I wanted to have this different feel-- there's trained fighters who are highly skilled but can be more predictable, vs untrained fighters who can be very strong in their own right. 

Making martial arts less effective when you're drunk/high also makes sense to me. 

I think mostly people have complained about both of these doing too much damage. I personally don't feel that but since you can't "choose" not to get good at brawling I was considering giving it a lower base damage compared to martial arts. Right now the damage from them both is equal, and stacks.

**Comment by Andrest07** [comment]
In regards to "gain opportunity" I think it'd be better if instead of an escape, the player has a chance to do even more damage than usual to an enemy depending on the player's martial arts skill in relation to the enemy's martial arts skill. So for example the higher the PC's skill is compared to the NPC, the higher chance the PC succeeds but if the PC's skill is lower, the lower the chance then when both PC and NPC are relatively equal, there's a chance it will be countered without consequence. Results from these checks can range from success which means extra damage, fail which means no damage to npc but also no damage to the pc and fail catastrophically which means potentially massive damage to the pc. If we figure out how to add martial arts and brawling skills to NPCs, we can have differing RNG rates to assign said skills depending on the location and who the NPC is. 

Furthermore, to balance Brawling I heavily suggest for it to have lower damage than martial arts, although it works for all situations, it shouldn't work as well as martial arts in a one on one fight. Hard to imagine a martial artist with a myriad of counters and techniques would lose against a street fighter even if he's been doing it for a long time.

Also about not being able to choose training brawl, maybe we can stop it's growth after it has reached B or something and anything beyond that would require training/advice from a named NPC who hangs out in the bar.

**Comment by noeinan** [comment]
I am still not really sure on adding special techniques to Martial Arts bc I don't want to cause too much overlap with Rukarin's mod and DoBL has done a lot about adding techniques and such. 

For a Run Away option, I like the idea of at least being able to do this, but thinking back... Won't this mean I have to write a new exit clause for literally every single encounter showing new text if you run away? Well, I guess we could add just one single generic line instead tho. That might be doable. 

I'm okay with Brawl doing a bit less damage. Maybe like, 3/4 damage of Martial Arts. 

For locking high level Brawl behind NPCs... I don't know about that. I don't like it... But maybe instead, like, you have to fight stronger enemies. So only certain "elite" enemies can help you increase Brawl after rank B.

**Comment by Andrest07** [comment]
I don't know about Rukarin's mod but afaik DoBL only adds fictional techniques but your call.

Generic line would definitely work, a lot of base game has generic exit passages.

Yep.

Sure, gonna need to figure out NPC generation with these skills as well tho.

**Comment by noeinan** [comment]
NPCs with Skills is def. a whole other ballpark. That will probably be a huge code rework, so far future. For now, we can at least do: 

-Reduce Brawl damage by 1/4
-Martial Arts less effective with more enemies
-Possibly reduced damage when drunk/high

**Comment by Andrest07** [comment]
Will look into it when I have the time ^^

**Comment by Andrest07** [comment]
Ok so, the 75% nerf still allowed Wub to 3 shot enemies with B brawling and no martial arts and with a 99% nerf Wub was still able to 7 shot enemies, oh boi

**Comment by Andrest07** [comment]
> 
> 
> Ok so, the 75% nerf still allowed Wub to 3 shot enemies with B brawling and no martial arts and with a 99% nerf Wub was still able to 7 shot enemies, oh boi

Seems to just be base game balance, we need better enemies lol

**Comment by Andrest07** [comment]
Combat-Rebalance branch:
- Reduces Brawling effectiveness by 25%
- Reduces Martial Arts effectiveness by 20% for every enemy (100%, 80%, 64%, 51.2% and so on)
- Alcohol, Drugs and Hallucinogens reduce global damage by 75% if all are at max (1000).

**Comment by Andrest07** [comment]
NPC-with-SKILLS branch:
- NPCs now have martial and brawl skills randomized from 1 to 800 if not manually assigned.
- Pain calcs to the PC now include enemy martial and brawl skills.
- <<martialdifficulty>> has been modified so if a third arg is included in usage, it will take the enemy's martial skill instead.
- PCs can also negate the enemy's martial skill with the above mentioned widget.

**Comment by noeinan** [comment]
This is in the new release.
